src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -25,21 +24,22 @@ internal class ProcessMetrics
     internal static readonly AssemblyName AssemblyName = typeof(ProcessMetrics).Assembly.GetName();
     internal static readonly Meter MeterInstance = new(AssemblyName.Name, AssemblyName.Version.ToString());
 
-    private const string MetricPrefix = "process.runtime.dotnet.";
+    private const string MetricPrefix = "process.";
```

Comment:
Curious - do we really need this const? (what benefit do we gain)

Reply:
I guess not much given currently it is just a 8 char string.... (not even for readability)